src/ManageCourses.Domain/Models/Provider.cs

```diff
@@ -45,6 +46,7 @@ public void UpdateWith(Provider provider)
         public string YearCode { get; set; }
         public string Scitt { get; set; }
         public string SchemeMember { get; set; }
+        public string AccreditingProvider { get; set; }
```

Comment:
May be worth storing this as a boolean, thoughts @misaka ?

Reply:
I thought that but as we have changed any other Y/N values to booleans I didn't bother

Reply:
i.e. `institution_type` and `copy_form_required`

Reply:
I'd definitely prefer these to be boolean, but if it means additional work in the .NET app I would probably hold off just for now. Switching this in Rails should be quite easy.

Reply:
Agreed @misaka. For all Y/N data values we should move to boolean but IMO its out of scope for this PR.